### DISCUSSION TITLE
Welcome to software-everywhere Discussions!

### DISCUSSION THREAD

**Discussion by seansund** [cloud-native-toolkit/software-everywhere#98] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by madhuraphatak27** [comment]
Hi Team,
The content here is quite technical. Could you please share some initial material to read or understand software everywhere
